src/components/NavigationSubpages/action-buttons.js

```diff
@@ -130,9 +130,7 @@ class ActionButtons extends Component {
 
           <div
             className={classes.rightElement}
-            onClick={() =>
-              window.open("http://localhost:3000/explore?", "_self")
-            }
+            onClick={() => window.open("http://localhost/explore?", "_self")}
           >
             <Reset className={classes.buttonIcon} /> <p>Zurücksetzen</p>
           </div>
```

Comment:
this will not work but also no need to hard-code this
javascript lets you extract the url (even in parts) and play with it 
check out https://developer.mozilla.org/en-US/docs/Web/API/Location